src/Microsoft.TestPlatform.ObjectModel/RunSettings/RunConfiguration.cs

```diff
@@ -250,6 +252,23 @@ public Architecture TargetPlatform
         }
     }
 
+    /// <summary>
+    /// Gets or sets the default platform that will be used for AnyCPU sources, or non-dll sources. Possible values are <see cref="Architecture"/> except for AnyCPU or Default.
+    /// </summary>
+    public Architecture? DefaultPlatform
+    {
+        get
+        {
+            return _defaultPlatform;
+        }
+
+        set
+        {
+            _defaultPlatform = value;
+            DefaultPlatformSet = true;
```

Comment:
Do we expect "null" to be set for the architecture? If not, then I would recommend to make this property an auto-property and remove `DefaultPlatformSet`.

Reply:
I am just doing what all the other settings are doing. We do expect the value to be null when the setting is not set. The whole logic of this class around nulls is not very clear, because e.g. _platform always has value, but is not always serialized.